installation.mdx

```diff
@@ -30,19 +30,23 @@ Awesome! Let's start creating ML-based applications
 
 Cerebrium caters to many different types of businesses and users so we have created abstractions that suit different users in different situations. Based on your use-case we recommend you look at the starting points below and examples in each sub-directory.
 
+#### When would you use Conduit over Cortex
+- The conduit is a very high level abstraction for deploying ML models and gives the end user a very specific way to deploy their ML models. Its best suited if you would like to deploy a model file or a HuggingFace model ID to production quickly with minimal setup. We recommend this for simple use cases that have simple pre/post processing or logging logic however don't worry, it can still scale to 10k requests per second.
```

Comment:
I like. A foil for you to compare styles with:

The conduit is a very high level abstraction for deploying ML models that gives the end user a very **streamlined and easy** way to deploy their ML models. Its best suited if you would like to deploy a model file or a HuggingFace model ID to production quickly with minimal setup. We recommend this for simple use cases that have simple pre/post processing or logging logic. **But don't let the simplicity fool you,** it can still scale to 10k requests per second.